packages/ozone/src/verification/service.ts

```diff
@@ -0,0 +1,192 @@
+import { Selectable } from 'kysely'
+import {
+  $Typed,
+  AppBskyActorDefs,
+  AtUri,
+  ToolsOzoneModerationDefs,
+  ToolsOzoneVerificationDefs,
+} from '@atproto/api'
+import { Database } from '../db'
+import { CreatedAtUriKeyset, paginate } from '../db/pagination'
+import { Verification } from '../db/schema/verification'
+
+export type VerificationServiceCreator = (db: Database) => VerificationService
+
+export class VerificationService {
+  constructor(public db: Database) {}
+
+  static creator() {
+    return (db: Database) => new VerificationService(db)
+  }
+
+  async create(
+    verifications: Pick<
+      Verification,
+      'uri' | 'issuer' | 'subject' | 'handle' | 'displayName' | 'createdAt'
+    >[],
+  ) {
+    // @TODO: We should probably handle errors here?
+    await this.db.transaction(async (tx) => {
+      return Promise.allSettled(
```

Comment:
Ideally use the transaction so that they all create or all fail together.

Reply:
Actually you could just do a single insert with all the verifications, that is probably simplest!